lucene/analysis/nori/src/java/org/apache/lucene/analysis/ko/tokenattributes/PartOfSpeechAttributeImpl.java

```diff
@@ -80,7 +80,7 @@ private String displayMorphemes(Morpheme[] morphemes) {
       if (builder.length() > 0) {
         builder.append("+");
       }
-      builder.append(morpheme.surfaceForm + "/" + morpheme.posTag.name() + "(" + morpheme.posTag.description() + ")");
+      builder.append(morpheme.surfaceForm).append("/").append(morpheme.posTag.name()).append("(").append(morpheme.posTag.description()).append(")");
```

Comment:
If you want to (micro)optimize even further you can use `#append(char)` instead of `#append(String)` for single characters.

Reply:
While true that it is a micro-optimization, I do believe it is a solid one. As in, a change that will perform better every single time. Not just in edge cases. And it is a very small improvement, but an improvement none the less. And this is only 1 subject.

Making these changes, running the tests, writing the PR took a good 30 minutes(mostly waiting on the tests). For something that isn't work on a feature. So I perfectly understand nobody wants to put their time in that and I just feel like seeing how much I can contribute in this fashion.